RiotSharp/StaticRiotApi.cs

```diff
@@ -96,42 +96,55 @@ public class StaticRiotApi : IStaticRiotApi
         /// </summary>
         /// <param name="apiKey">The api key.</param>
         /// <returns>The instance of StaticRiotApi.</returns>
-        public static StaticRiotApi GetInstance(string apiKey)
+        public static StaticRiotApi GetInstance(string apiKey, bool useCache = true, 
+            TimeSpan slidingExpirationTime = default(TimeSpan))
         {
             if (instance == null || 
                 Requesters.StaticApiRequester == null ||
                 apiKey != Requesters.StaticApiRequester.ApiKey)
             {
-                instance = new StaticRiotApi(apiKey);
+                instance = new StaticRiotApi(apiKey, useCache, slidingExpirationTime);
             }
             return instance;
         }
 
-        private StaticRiotApi(string apiKey)
+        private StaticRiotApi(string apiKey, bool useCache = true, TimeSpan slidingExpirationTime = default(TimeSpan))
         {
-            Requesters.StaticApiRequester = new Requester(apiKey);
+            Requesters.StaticApiRequester = new RateLimitedRequester(apiKey, new Dictionary<TimeSpan, int>
+            {
+                { new TimeSpan(1, 0, 0), 10 }
+            });
             requester = Requesters.StaticApiRequester;
-            cache = new Cache();
+            if (useCache)
+                cache = new Cache();
+            else
+                cache = new PassThroughCache();
+
+            if (slidingExpirationTime != default(TimeSpan))
+                SlidingExpirationTime = slidingExpirationTime;
```

Comment:
You can't do that, because `new TimeSpan()` is not a compile-time constant. However I do set it a default in line 89.